egov-persister/hearing-persister.yml

```
@@ -102,9 +106,10 @@ serviceMaps:
             - jsonPath: $.hearing.notes
             - jsonPath: $.hearing.auditDetails.lastModifiedBy
             - jsonPath: $.hearing.auditDetails.lastModifiedTime
+            - jsonPath: $.hearing.hearingDurationInMillis
             - jsonPath: $.hearing.id 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)